esmvalcore/config-developer.yml

```diff
@@ -26,6 +26,20 @@ CMIP6:
     DKRZ: '{activity}/{institute}/{dataset}/{exp}/{ensemble}/{mip}/{short_name}/{grid}/{latestversion}'
     ETHZ: '{exp}/{mip}/{short_name}/{dataset}/{ensemble}/{grid}/'
   input_file: '{short_name}_{mip}_{dataset}_{exp}_{ensemble}_{grid}*.nc'
+  catalogs:
+    DKRZ:
```

Comment:
how are these catalogs produced? We should get one done at JASMIN too, if I get told how to make one :grin: